design/volume-snapshot-data-movement/volume-snapshot-data-movement.md

```diff
@@ -666,6 +666,9 @@ At present, VBDM doesn't support recovery, so it will follow the second rule.
 ## Kopia For Block Device
 To work with block devices, VGDP will be updated. Today, when Kopia attempts to create a snapshot of the block device, it will error because kopia does not support this file type. Kopia does have a nice set of interfaces that are able to be extended though.
 
+**Notice**
```

Comment:
can you change the changelog of previous PR
so that changelog of release is clear

Reply:
Sure. I will modify the v1.12.1's changelog too.

Reply:
Modified in the unreleased changelog and the v1.12.1 release note.
https://github.com/vmware-tanzu/velero/releases/tag/v1.12.1